test/plugin/scenarios/webflux-scenario/config/expectedData.yaml

```diff
@@ -193,6 +193,66 @@ segmentItems:
         parentSpanId: 8, parentTraceSegmentId: not null, parentServiceInstance: not
           null, parentService: not null, traceId: not null}
       skipAnalysis: 'false'
+  - segmentId: not null
+    spans:
+    - operationName: /testcase/webclient/test1
+      operationId: 0
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Http
+      startTime: not null
+      endTime: not null
+      componentId: 67
+      isError: false
+      spanType: Entry
+      peer: ''
+      skipAnalysis: false
+      tags:
+      - {key: url, value: 'http://localhost:18080/testcase/webclient/test1'}
+      - {key: http.method, value: GET}
+      - {key: status_code, value: '200'}
+      refs:
+      - {parentEndpoint: /projectA/testcase, networkAddress: 'localhost:18080', refType: CrossProcess,
+        parentSpanId: 9, parentTraceSegmentId: not null, parentServiceInstance: not null, parentService: not null, traceId: not null}
+  - segmentId: not null
+    spans:
+    - operationName: /testcase/annotation/healthCheck
```

Comment:
Why health check in the expected span?

Reply:
Thanks for quick reply. I use it for webclient testcase. The following code：
```
@RequestMapping("/testcase/webclient/test1")
    public String test1() {
        Mono<String> resp = WebClient.builder().exchangeStrategies(ExchangeStrategies.builder()
                .build())
                .build()
                .get().uri("http://" + hostBAddress + "/testcase/annotation/healthCheck")
                .retrieve().bodyToMono(String.class);
        resp.subscribe();
        return "ok";
    }
```
Would you mind sharing your tips here?

Reply:
Don't access health check by codes. It is designed for the test framework. If you need a uri to access, add a new one.

Reply:
Thanks. fixed at last commits.